lib/TeX-lab.js

```diff
@@ -26,7 +26,7 @@ const Lab = window.Lab = {
   Typeset() {
     let TeX = this.tex.value;
     let math = new LabMathItem(TeX,tex);
-    math.setMetrics(16,8,1000000,100000,1);
+    math.setMetrics(16,8,16*20,100000,1);
```

Comment:
Whitespace.

Reply:
These are not in files that are part of MathJax itself -- they are that javascript that controls the labs.  Those files have not followed the tslint requirements (and have  lots of other lint issues), so I'm going to leave these for now.  We can do a full clean up on those files if we decide to keep them permanently.